cms/static/sass/views/_container.scss

```diff
@@ -55,6 +55,14 @@
             }
           }
         }
+
+        .container-unit-access {
+          @include font-size(14);
+          line-height: 1.5;
+          white-space: normal;
+          color: $black;
```

Comment:
Ah yeah I meant to bring this up.  I agree there should be a consistency between the colors but there was an accessibility issue with using the other color so I switched to black as a default.